### DISCUSSION TITLE
Adição das Desvantagens na Caverna do Saber

### DISCUSSION THREAD

**Discussion by SloppyMaster11** [YuriAlessandro/gerador-ficha-tormenta20#351] [Ideias]
Um conteúdo interessante que falta no índice.
Presente na DB#156

**Comment by YuriAlessandro** [comment]
Boa, vou adicionar. Eu acho que ainda não tinha ido tão longe para trás nas versões (comecei das mais novas e fui até a 169)

**Comment by YuriAlessandro** [comment]
Adicionado, muito obrigado. Aproveitei e dei uma olhada da DB#156 até a DB#168 então agora está bem mais atualizado.

**Comment by fstomazini** [comment]
Consta no app desvantagens na DB#152 enquanto o correto seria na DB#156
![Captura de Tela 2023-07-18 às 12 10 21](https://github.com/YuriAlessandro/gerador-ficha-tormenta20/assets/44268872/c408f60a-39b3-4c15-9334-963e1498bbef)

**Comment by YuriAlessandro** [comment]
o amigo @caironoleto corrigiu isso através de #354 

já publiquei
